transcendence-app/types/github-like-avatar-generator.d.ts

```diff
@@ -0,0 +1,20 @@
+type GenerateAvatarParams = {
+  blocks: number;
+  width: number;
+  fromPixels?: Array<string>;
+};
+
+type GenerateAvatarReturnType = {
+  base64: string;
+  pixels: Array<string>;
```

Comment:
We have been changing Array<string> to string[], every now and then. I was sold on the idea of using this(by one of you two), but I was argued against (by the other one of you two), and now I'm not sold anymore. I don't think at this point Array<string> is our preferred choice across the project. Please. Let's stick to one convention.

Reply:
Done :+1: